lib/customer.rb

```diff
@@ -0,0 +1,66 @@
+require "csv"
+
+class Customer
+  attr_reader :cust_id
+  attr_accessor :cust_email, :cust_address
+
+  def initialize(cust_id, cust_email, cust_address)
+    @cust_id = cust_id
+    @cust_email = cust_email
+    @cust_address = cust_address
+  end
+
+  def id
+    return @cust_id
+  end
```

Comment:
You don't need to hand-write these getter methods if you use the `attr_reader` and `attr_accessor` helpers above.